sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/util/ResolveDefaultColumnsUtil.scala

```diff
@@ -320,6 +319,29 @@ object ResolveDefaultColumns extends QueryErrorsBase
     coerceDefaultValue(analyzed, dataType, statementType, colName, defaultSQL)
   }
 
+  /**
+   * Analyze EXISTS_DEFAULT value.  This skips some steps of analyze as most of the
+   * analysis has been done before.
+   */
+  private def analyzeExistingDefault(field: StructField): Expression = {
```

Comment:
nit: we might name this `analyzeExistenceDefaultValue` as we use the term "existence default value" to differentiate between "current default value". These are two separate metadata entries for the `StructField`, where the former gets filled in at scan time for each row where the corresponding column value is not present in storage (e.g. the Parquet vector is absent) and the latter gets filled in at DML time only.

Reply:
Done.